packages/react-ui-ag/src/Gmap/markerIcons.js

```diff
@@ -10,5 +10,5 @@ export const markerBlackDotIcon = () => ({
 
 export const markerBlackDotIconWithBalloon = () => ({
   url: "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='38' height='65' viewBox='0 0 38 65'%3E%3Cdefs%3E%3Cpath id='b' d='M12.598 31.638C5.396 30.078 0 23.669 0 16 0 7.163 7.163 0 16 0s16 7.163 16 16c0 7.248-4.82 13.37-11.43 15.338l-4.054 8.92-3.918-8.62z'/%3E%3Cfilter id='a' width='137.5%25' height='129.8%25' x='-18.8%25' y='-12.4%25' filterUnits='objectBoundingBox'%3E%3CfeMorphology in='SourceAlpha' operator='dilate' radius='1' result='shadowSpreadOuter1'/%3E%3CfeOffset dy='1' in='shadowSpreadOuter1' result='shadowOffsetOuter1'/%3E%3CfeGaussianBlur in='shadowOffsetOuter1' result='shadowBlurOuter1' stdDeviation='1.5'/%3E%3CfeComposite in='shadowBlurOuter1' in2='SourceAlpha' operator='out' result='shadowBlurOuter1'/%3E%3CfeColorMatrix in='shadowBlurOuter1' values='0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0'/%3E%3C/filter%3E%3Ccircle id='d' cx='8' cy='8' r='8'/%3E%3Cfilter id='c' width='151.3%25' height='151.3%25' x='-25.6%25' y='-19.4%25' filterUnits='objectBoundingBox'%3E%3CfeMorphology in='SourceAlpha' operator='dilate' radius='.6' result='shadowSpreadOuter1'/%3E%3CfeOffset dy='1' in='shadowSpreadOuter1' result='shadowOffsetOuter1'/%3E%3CfeGaussianBlur in='shadowOffsetOuter1' result='shadowBlurOuter1' stdDeviation='1'/%3E%3CfeComposite in='shadowBlurOuter1' in2='SourceAlpha' operator='out' result='shadowBlurOuter1'/%3E%3CfeColorMatrix in='shadowBlurOuter1' values='0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0'/%3E%3C/filter%3E%3C/defs%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cg transform='translate%283 2%29'%3E%3Cuse fill='%23000' filter='url%28%23a%29' xlink:href='%23b'/%3E%3Cuse fill='%23D32526' stroke='%23FFF' stroke-width='2' xlink:href='%23b'/%3E%3C/g%3E%3Cg fill='%23000' transform='translate%2811 46%29'%3E%3Cuse filter='url%28%23c%29' xlink:href='%23d'/%3E%3Cuse stroke='%23FFF' stroke-width='1.2' xlink:href='%23d'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E%0A",
-  anchor: new window.google.maps.Point(19, 56),
+  anchor: new window.google.maps.Point(19, 54),
```

Comment:
What do you think about making it possible to override the anchor coordinates in case this needs to change in the future?

Reply:
You can already do this by overriding the whole function if you need to. I was thinking about pulling these out of react-ui completely and just have the app handle it instead.